storage/bucket.go

```diff
@@ -1174,6 +1175,8 @@ type BucketAttrsToUpdate struct {
 	RPO RPO
 
 	// If set, updates the autoclass configuration of the bucket.
+	// Library users should provide the updated autoclass configuration
+	// with the enabled field explicitly set.
```

Comment:
I'd say something like:

`To disable autoclass on the bucket, set to an empty &Autoclass{}. To update the configuration for Autoclass.TerminalStorageClass, Autoclass.Enabled must also be set to true.`

Reply:
This is much clearer, thanks! Updated.